LISA-High-Level-view.xml

```diff
@@ -0,0 +1 @@
+<mxfile userAgent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/603.3.8 (KHTML, like Gecko) Version/10.1.2 Safari/603.3.8" version="7.2.9" editor="www.draw.io" type="device"><diagram id="8c92e61c-5d4b-58e6-bfb7-0b78ab075018" name="Page-1">7Vptb5s6FP41fEwEOBDysc3tukmrFt1UutunyYBDrBqcGeel99dfG2wC2Omi3SRLp1RqZc4xxj7Pw/FzTB0wzXePDK6WTzRFxPHddOeAvxzfn4wm4q80vNaGIIhqQ8ZwWpu8vWGO/0XK6CrrGqeo7HTklBKOV11jQosCJbxjg4zRbbfbgpLuU1cwQ4ZhnkBiWv/BKV/W1sgP9/aPCGdL/WQvVAuOYfKSMbou1PMcHyyqn9qdQz2WWmi5hCndtkzgwQFTRimvW/luiogMrQ5bfd+HA95m3gwV/Kgb1Dw2kKyRnnI1Mf6qg1EtB8kbXAfcb5eYo/kKJtK7FegL25LnRFx5ollyRl/QlBLKqruBW/0IzwIT0rKrsAg7LfgHmGMiufIRkQ3iOIHKoajh+eraNjAkOCuELRFrRsJ5bwZBxWWDGEe7lkkF5RHRHHH2Kroob6DiovjrT9QI2z0bGgyXbSZMlBEqBmbN0HsUREMBYQdlcsPEhgkY90FxDVAAsIDSGP8PKL4ByjODmJTivofdiqGyPDFIYRKheGGClEIULZKrBSkCHZBGrgmSZ3tzwhNgNDIw4hIjEfOXQbKEPBZp9QaSBaRxdDmQxj/PbqhI7+QuLpdNYFnipIo9ZNw0t9HRm6lrgwoAFdHao/f18BQgoR3mX9UsZPubnMQwkFeFCM9XNafqYu87iKRYaYZ4J++gtKNYTGhb0AUW5LSNIQI53nR1jg1O9YQZxWJyDXMaBmjm9FNrSdcsQequtvDoDQT83kBBb6A6BsZAFbuaZR9FOM9M3W8xrqAFuhpeNezZk+mbJpPinHIp1mkGzhDDIkYyY1S93yHXRqNulgLRL3JNbEKdgQL3fFwDBtceKc0EzXx3vkIoWYrG3ezTibehNIzDILQKOj+53m2op7LHF5QKnqkV5rRYmyJOLkeEinyGMSIzWmKOqQxEjtNU9rnXHe5UhBqHAZhODX6dU1byCfkukxX0cEHoVkgUxoe0GMj68jtDCySCKylwQXVxWOGQXgBOyYTQ677pvoUJroUJ/YzwS0yIbky4HiZ4vSIvvCQTzMJ7XtBtTF9PnK7fZ9Xg9bSfZ8PmXPlaj/sHiThPX/S1ml3ZvaniainUpXFb2ekkdyXSzgM9RdZX/8dKOy+6mLRrXph3zEDPMYrQtxlo1LPHMRDcGHgOBpqFLMElHJQcFikkgnADxw+JCNR9ijeimcnmhuIEDRKa56KX9sdMu7VFPLx1023Dc8xjMqssPduGZ1aSFdgrss5wUQ4EmkIbMgvgT5VHLr7qKseUMYGJlGvlDe/DH31A0MH7omfXfmDHO6Y7AxyxHv4TBCI/BqHlRCANUJSObAE8WL3YwD/F2zXuFn22b2xAfwtrR9s/RbTNU+h+Ku3lzT8RgqCPgG8g0KjJkyNglt0VAmtsSWl3RbYmUCQ7dyR+F4xWOzBcrW7Z7I1sFv6+3QuY5drz89ypDkHhi3Xbmn2afnF8MbI7o4S8DofDI8G9nctc4lymd1YLLB/fz3YwA8zSa/9ht0Q/1ojISJqc+vL3k8wXVGaO8gcRr71opJDDGJboljuOzh2jcXi23CEu9//vVFdF+/8pAw//AQ==</diagram></mxfile>
```

Comment:
maybe it will be good to create a folder `docs` and put it in here with also a png file representing this